script/watch-tests.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-console */
```

Comment:
This is a simple rename from what used to be the `watch.js`. I did have to fix some linting errors, though, because eslint yelled at me. Not sure why this was never fixed up before. :thinking: